dtfs-central-api/src/v1/controllers/portal/facility/get-amendment.controller.ts

```diff
@@ -18,7 +18,7 @@ export const getAmendment = async (req: GetAmendmentRequest, res: Response) => {
     const amendment = await TfmFacilitiesRepo.findOneAmendmentByFacilityIdAndAmendmentId(facilityId, amendmentId);
 
     if (!amendment || amendment.type !== AMENDMENT_TYPES.PORTAL) {
```

Comment:
```suggestion
    if (!amendment?.type !== AMENDMENT_TYPES.PORTAL) {
```

Reply:
@abhi-markan I sometimes think there's a benefit to having something written out more explicitly - while the suggested change might also be correct and shorter, it took me longer to read and understand what's going on.

In this case, I'd prefer it if Mark sticks to his original option so it's easier to read on first glance. 

What do you think?

Reply:
I'm happy with either :)